.azure-pipelines/template-variables.yml

```diff
@@ -4,6 +4,7 @@ variables:
   SONIC_SLAVE_DOCKER_DRIVER: 'overlay2'
   SONIC_BUILD_RETRY_COUNT: 3
   SONIC_BUILD_RETRY_INTERVAL: 600
+  OVERRIDE_BUILD_OPTIONS: ''
```

Comment:
Please check if the variable in the build environment will override the default setting?